index.js

```diff
@@ -19,34 +20,51 @@ const tokenPayload = {
   "client_secret": github_client_secret
 };
 const requestAccessTokenOptions = {
-  url: githubApi.githubTokenUrl,
+  url: githubConfig.githubTokenUrl,
   method: 'POST',
   json: true,
   headers: {
     Accept: 'application/json'
   }
 };
 
-githubApi.githubAuthUrl += '?client_id=' + github_client_id
+githubConfig.githubAuthUrl += '?client_id=' + github_client_id
 
 app.use(express.static(path.join(__dirname, 'public')));
 app.engine('ejs', ejs.renderFile);
+app.use(session({
+  secret: 'super secret keyboard cat'
+}));
```

Comment:
Probably not too secure... but I'm not storing anything *that* critical I don't think......... 🐑